Remove black border for share card image

schamp opened the pull request:
And move updated date into a subheader

schamp commented on src/components/SocialLocationPreview/SocialLocationPreviewMap.tsx:
```diff
@@ -34,6 +36,7 @@ const SocialLocationPreview = (props: {
   return (
     <Wrapper border={props.border}>
       <MapHeaderHeader>US COVID Risk &amp; Vaccine Tracker</MapHeaderHeader>
+      <LastUpdatedWrapper>Updated {lastUpdatedDateString}</LastUpdatedWrapper>
```
Note this changes the displayed date from the current date when the image was generated to the date the model was updated, which I believe to be the correct value to show.  If I have that wrong, LMK.

schamp replied:
per Igor in ticket, "match the homepage", which this does.

schamp commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Remove black border for share card image](https://trello.com/c/IDTQ1xRi/1268-remove-black-border-for-share-card-image)

mikelehen reviewed (changes requested):
Thanks for tackling this!  Unfortunately it probably needs a bit more work.  The current share card looks like this:

![image](https://user-images.githubusercontent.com/206364/121835379-06668c80-cc86-11eb-812f-0e657aab6048.png)

Your updated one looks like this:
![image](https://user-images.githubusercontent.com/206364/121835395-0ebec780-cc86-11eb-9f75-87f4dce6e646.png)

The image is intentionally 1200x630 to optimize the resolution for Twitter / Facebook / etc. sharing.  So the updated share card end up with a bunch of whitespace...  I think we want the ShareCardWrapper to be as big as possible and probably remove the drop shadow so that it's just a uniform white background in the image.

The actual share image generation happens via code that runs on-demand via Firebase Functions by visiting `/internal/share-image/` in a browser and then captures a screenshot.  For reference, that code lives [here](https://github.com/covid-projections/covid-projections/blob/5074966bce8c468e2ab32026d0a34b24466815e2/firebase/functions/src/screenshots.ts#L33) but the important part for testing locally is to just follow this [README](https://github.com/covid-projections/covid-projections/blob/develop/firebase/README.md).

Let me know if this is too much trouble to deal with.

schamp commented:
Thanks for explaining, Michael, I wasn't aware of those constraints.  I'll take another stab at it.

schamp commented:
updated: 
![image](https://user-images.githubusercontent.com/351548/122620875-ed285c00-d048-11eb-9fbf-a28ad7027956.png)
(ignore black bar at bottom).

@mikelehen I scaled up and removed the border, LMKWYT

mikelehen commented on src/components/SocialLocationPreview/SocialLocationPreview.style.tsx:
```diff
@@ -7,7 +7,9 @@ export const Wrapper = styled.div<{
 }>`
   box-shadow: ${props =>
-    props.border ? 'none' : '0px 12px 40px rgba(0, 0, 0, 0.2)'};
+    props.noShadow || props.border
```
Should there be parens around this?  As-is, I think you'll end up with `box-shadow: true` if `props.noShadow` is true? But I could easily be confused.

schamp replied:
Now you've confused me and I'm not sure but parens wouldn't hurt so I'll add them.

mikelehen replied:
Looks like you didn't add them, which might mean that I was confused and `prettier` removed them because they're not necessary. 🤷

schamp replied:
yeah I added them and prettier re-removed them so :shrug:

mikelehen reviewed (changes requested):
I think the share image looks good but it looks like this also tweaked the preview image in the share widget at the bottom of the homepage (removing the last updated text and the drop-shadow).  Not sure if both those changes are intentional? 

Before:
![image](https://user-images.githubusercontent.com/206364/122624931-5911c100-d057-11eb-8954-b966312d0b90.png)

After:
![image](https://user-images.githubusercontent.com/206364/122624941-63cc5600-d057-11eb-9e9a-7ec7b4a33b75.png)

schamp commented:
> I think the share image looks good but it looks like this also tweaked the preview image in the share widget at the bottom of the homepage (removing the last updated text and the drop-shadow). Not sure if both those changes are intentional?
> 
> Before:
> ![image](https://user-images.githubusercontent.com/206364/122624931-5911c100-d057-11eb-8954-b966312d0b90.png)
> 
> After:
> ![image](https://user-images.githubusercontent.com/206364/122624941-63cc5600-d057-11eb-9e9a-7ec7b4a33b75.png)

That change was not intentional, I'll see if I can sort it out.  I had a look, but didn't notice the difference, thanks for catching it.

schamp commented:
Fixed, I think.  I'm not sure how much I like the preview being different from the actual share image, but it looks good in this context, so I think it's fine:
![image](https://user-images.githubusercontent.com/351548/122625692-4bf6d100-d05b-11eb-9c2b-a96ae8a8bd3c.png)

mikelehen reviewed (approved):
Thanks! Looks good.  I agree it's a bit weird that the preview doesn't match the result, but it never has...   🤷
